### PR TITLE
111827055 - Add feature to create haiku lines

### PR DESCRIPTION
**Pull request by thomas-jung**
Add next line using lines#new form. View conditional that limits form up the 3rd line.

### CODE REVIEW

**Comment by kangkyu** [comment]
I think we could add spec for line count max 3. (Later?)
How about clean up all other methods in lines_controller? They are not going to be used anyways.

File: spec/factories/haikus.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
+  params = { 0 => { :content => 'first line' } }
   factory :haiku do
-
+    lines_attributes params
   end
```
Comment (kangkyu) [inline spec/factories/haikus.rb]: Wow how did you do this! It should be on the documentation

**Comment by charliemcelfresh** [comment]
@thomas-jung Let's get this in :hamburger: but we need to 1) put a flash message where there is an error, 2) put tests in for that -- so will you make one ticket for adding those two things?

**Comment by thomas-jung** [comment]
I wrote ticket for flash message and tests. I also wrote ticket for the max of 3 lines validation.

**Pull request merged**